src/main.js

```diff
@@ -218,49 +218,58 @@ app.whenReady().then(async () => {
 		}
 	})
 
+	const trustedFingerprints = getAppConfig('trustedFingerprints') ?? []
 	app.on('certificate-error', (event, webContents, url, error, certificate, callback) => {
 		event.preventDefault()
 
 		if (isLinux) {
-			return callback(process.env.NODE_ENV !== 'production')
+			if (trustedFingerprints.includes(certificate.fingerprint)) {
```

Comment:
Let's always get the `trustedFingerprints` from the app config in the event handler.

Currently, there is no error from getting the config value only once, because `trustedFingerprints` variable is changed only here. But if there is another source of changing the config, here we'll have an outdated value. For example, we may add a fingerprint reset option in the future.

`getAppConfig` is cheap, it uses in-memory cache.